scripts/xe-reset-networking

```diff
@@ -120,26 +123,47 @@ if __name__ == "__main__":
 
 	# Determine IP configuration for management interface
 	options.mode = options.mode.lower()
-	if options.mode not in ["dhcp", "static"]:
-		parser.error('mode should be either "dhcp" or "static"')
+	if options.mode not in ["none", "dhcp", "static"]:
+		parser.error('mode should be either "none", "dhcp" or "static"')
+		sys.exit(1)
+
+	options.mode_v6 = options.mode_v6.lower()
+	if options.mode not in ["none", "autoconf", "dhcp", "static"]:
+		parser.error('mode-v6 should be either "none", "autoconf", "dhcp" or "static"')
+		sys.exit(1)
+
+	if options.mode == "none" and options.mode_v6 == "none":
+		parser.error("Either mode or mode-v6 must be not 'none'")
 		sys.exit(1)
 		
 	if options.mode == 'static' and (options.ip == '' or options.netmask == ''):
 		parser.error("if static IP mode is selected, an IP address and netmask need to be specified")
 		sys.exit(1)
-	
+  
+	if options.mode_v6 == 'static':
+		if options.ipv6 == '':
+	 		parser.error("if static IPv6 mode is selected, an IPv6/CIDR address need to be specified")
+		elif options.ipv6.find('/') == -1:
+			parser.error("Invalid format: IPv6 must be specified with this format: <IPv6>/<CIDR>")
```

Comment:
Terminology: the CIDR notation of an IP address is the whole `x/y` thing. The number after the `/` is the prefix length, which is a more compact way of denoting the netmask.

Reply:
Thanks for the clarification, I've changed the text :)